src/fciqmc_col.jl

```diff
@@ -1,5 +1,15 @@
 # third level: `fciqmc_col!()`
 
+step_stats(v, n) = step_stats(StochasticStyle(v), n)
+function step_stats(s::StochasticStyle, ::Val{N}) where N
+    if N == 1
+        return step_stats(s)
+    else
+        names, stats = step_stats(s)
+        return names, MVector(ntuple(_ -> stats, Val(N)))
+    end
+end
+
```

Comment:
This seems an elegant solution. Now new `StochasticStyle`s require a new method for `step_stats(s)` and `fciqmc_col!`. I suggest to add an explanation in the docstring of `StochasticStyle`.